osu.Game/Rulesets/Mods/ModFlashlight.cs

```diff
@@ -119,21 +118,33 @@ private void load(ShaderManager shaderManager)
 
             private readonly IBindable<bool> isBreakTime = new BindableBool();
 
+            /// <summary>
+            /// Returns the aspect ratio-adjusted size of the flashlight.
+            /// This ensures that the size of the flashlight remains independent of ruleset-specific aspect ratio adjustments.
```

Comment:
This comment is only true for taiko's override of this? It cannot be moved into the xmldoc of the general abstract method. Abstract method xmldocs are supposed to tell *consumers*, i.e. ruleset developers, what they should be overriding the method to.

Reply:
You are right. maybe:

Creates the draw flashlight area.
e.g: mania draws on the entire width of the playfield and ajusts the height by a given size, so this should return Vector2(DrawWidth, size).